tests/customer-resource-visibility-test.js

```diff
@@ -1,10 +1,10 @@
-import { beforeEach, afterEach, describe, it } from '@bigtest/mocha';
+import { beforeEach, describe, it } from '@bigtest/mocha';
 import { expect } from 'chai';
 
 import { describeApplication } from './helpers';
-import CustomerResourceShowPage from './pages/customer-resource-show';
+import CustomerResourceShowPage from './pages/bigtest/customer-resource-show';
 
-describeApplication('CustomerResourceVisibility', () => {
+describeApplication.only('CustomerResourceVisibility', () => {
```

Comment:
Oops, don't want the `.only()` checked in.

Reply:
Dang it! My bad hahaha